bootique-shiro-web-jwt/src/main/java/io/bootique/shiro/web/jwt/token/JwtToken.java

```diff
@@ -0,0 +1,19 @@
+package io.bootique.shiro.web.jwt.token;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class JwtToken {
+
+    private Set<String> roles;
+
+    public Set<String> getRoles() {
+        return roles == null ? Collections.emptySet() : roles;
+    }
+
+    void setRoles(List<String> roles) {
```

Comment:
No reason for this object to be mutable. All the initialization should happen in constructor